docs/configuration.md

```diff
@@ -26,6 +26,41 @@ variables.
 - Applies to: listener, worker, cron.
 - Format: string.
 
+###  CONFIG_REDIS_USERNAME
+
+- Redis ACL user name
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: string.
+
+###  CONFIG_REDIS_PASSWORD
+
+- Redis ACL password
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: string.
+
+###  CONFIG_REDIS_CA_FILE
+
+- Certification authority certificate Redis should trust to accept TLS connections
```

Comment:
```suggestion
- Certification authority to validate Redis server TLS connections with
```

btw usually software also supports `CA_DIR` option but maybe for the use case we won't need. I haven't checked how we ensure trust with HTTPS connections that we use in 3scale.

Reply:
Yes, that's also supported by `redis-client`, I might implement it in the future... maybe...

Reply:
The more important point of my comment is to figure out how the feature is to be used with the operator on-prem and make sure it suits :)

It's not presently clear to me, how are users expected to supplement this.

Reply:
Yes, it's good to provide a CA_DIR env var.

Reply:
The dir thing may or may not be practical depending on the way users are supposed to supply that to the pod.